modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -3,76 +3,112 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the processes for removing an Anypoint Platform PCE node from active service, and returning it to service when ready.
+== Before You Begin
 
-== Notes
+* Set up and test the backup procedure on all clusters after you complete installation.
+* Perform a backup before attempting node maintenance to preserve data in the event of a failure.
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+The following steps must be performed on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. The entire process must be completed on each node before the next node is addressed. Verify the health status of all cluster components and applications after each node update before proceeding with the next node.
 
-Before any maintenance or restart of a node can be performed, it must be removed from the cluster and its workload reassigned to the other nodes in the cluster.
+This approach assumes that multiple masters are running.
 
-. Check if the firewalld service is running and enabled:
-+
-----
-service firewalld status
-----
-. If firewalld is running, stop, disable, and mask it:
+=== Verify Cluster Status
+
+Verify that the cluster is healthy before starting the process. Otherwise, many of the assumptions made in this process are not valid.
+
+. Verify the output from the following commands:
 +
-----
-sudo systemctl stop firewalld
-sudo systemctl disable firewalld
-sudo systemctl mask firewalld
-----
-. Retrieve the *name* of target node:
+```
+gravity status
+etcdctl cluster-health
+gravity get pod -o wide--all-namespaces | grep -vi Running
+gravity get deployment --all-namespaces
+gravity get daemonset --all-namespaces
+```
+. Review log entries or status message that indicate the possibility of an unhealthy node.
```

Comment:
@eldios how are customers suppose to run the validations?
Is this step really necessary if they are already following the process?
Customers don't have access to the runbook...
what does it mean for them to `Verify that replication works and is healthy in any replicated software (like Stolon and Cassandra).`?

Reply:
The point is that if Pods are running but, as an example, Stolon is shouting errors in the logs, then it's not really a green light situation.

In my opinion is important to tell customers to be careful about all components, but it's up to you nice people to decide what your customers should and should not know.

I prefer to be on the safe/cautious side.